data/workflows/en/01_bootstrap_project.md

```diff
@@ -2,61 +2,59 @@
+As shown above, `dune init` accepts a _kind_, `NAME`, and optional `PATH` to scaffold new code. Let's try it out:
 
-To bootstrap a new project with `dune init`, run:
+```sh
+dune init project hello ~/src/ocaml-projects
 
-```
-dune init proj hello my_project/
+Success: initialized project component named hello
 ```
 
-Where proj is the kind of project to initialize. Here we want to generate an entire project, so we use `proj`. `hello` is the name of the project and `my_project/` is the path where the project will be generated.
+In the above example, we use:
 
-`dune init proj` does not generate a `dune-project` for you, so you need to go in the generated project and create one:
+- "project" as the _kind_
+- "hello" as the _name_, and
+- "~/src/ocaml-projects" as the _path_ to generate the content in
 
-```
-echo "(lang dune 2.0)" > dune-project
-```
+The the `project` _kind_ creates a `library` in `./lib`, an `executable` in `./bin`, and links them together in `bin/dune`. Additionally, the command creates a test executable and an opam file.
 
-At this point, you can build the project and run the binary:
+```sh
+tree ~/src/ocaml-projects/hello/
 
+/home/user/src/ocaml-projects/hello/
+├── bin
+│   ├── dune
+│   └── main.ml
+├── hello.opam
+├── lib
+│   └── dune
+└── test
+    ├── dune
+    └── hello.ml
 ```
-dune build
-dune exec bin/main.exe
-```
-
-`dune init` is the quickest way to get a working OCaml project and start hacking, but you may need a bit more, for instance:
-
-- How to setup the IDE
-- How to setup the CI/CD
 
-Or you may be looking for the best way to get started with a specific kind of project:
-
-- A library
-- A command line interface
-- A web application
-
-If that's the case, we recommend using `spin`, the OCaml project generator. Spin comes with official templates for common project types. The official templates will get you up and running with everything you need to get a productive development environment, including the IDE setup, the CI/CD, the code formatter, the unit tests, etc.
-
-You can install `spin` with opam: `opam install spin`.
+At this point, you can build the project and run the binary:
 
-Once it's installed, you can list the available templates with `spin ls`. For the purpose of this workflow, we'll use `bin` that bootstraps a project with an executable:
+```sh
+cd /home/user/src/ocaml-projects/hello/
+dune exec bin/main.exe
 
+Hello, world!
 ```
-spin new bin my_project/
-```
-
-This will take some time, because Spin will install all of the dependencies in a new opam local switch (a.k.a local sandbox), which needs to compile an OCaml compiler.
 
-Once the project is generated, you can run the executable with `make start`.
+Thus, `dune init` can rapidly scaffold new projects, with minimal content. It can also be used to add incremental components (kinds) to existing projects.
```

Comment:
If I understand this correctly should it be: 

```suggestion
Thus, `dune init` can rapidly scaffold new projects, with minimal content. It can also be used to add components (kinds) incrementally to existing projects.
```